lib/v2/bangumionline/index.js

```diff
@@ -0,0 +1,25 @@
+const got = require('@/utils/got');
+
+module.exports = async (ctx) => {
+    const url = 'https://bangumi.online/api/home';
+
+    const response = await got({
+        method: 'post',
+        url,
+        headers: {},
+    });
+
+    const list = JSON.parse(response.body).data.list;
```

Comment:
```suggestion
    const response = await got(url);

    const list = response.data.data.list;
```

Reply:
Using `const response = await got(url);` will return response code 404.

Reply:
`const response = await got.post(url);`

Reply:
Thank you ~